app/controllers/grantees_controller.rb

```diff
@@ -0,0 +1,73 @@
+class GranteesController < ApplicationController
+  before_action :set_grantee, only: %i[show edit update destroy]
```

Comment:
[suggestion] since we only have one route/method here currently, i'd prefer this not be abstracted out, and instead be set within the `show` method.

it also would be good to have the ID that is passed through the params be the grantee_id, rather than having it be random